src/context/SubaccountContext/SubaccountContext.tsx

```diff
@@ -303,8 +297,7 @@ function useSubaccountCustomSigners() {
   const privateKey = useSubaccountPrivateKey();
 
   return useMemo(() => {
-    const publicRpc = getRpcUrl(chainId);
-    const fallbackRpc = getFallbackRpcUrl(chainId);
+    const { default: publicRpc, fallback: fallbackRpc } = getBestRpc(chainId);
```

Comment:
I think this problem was before but now it seems it would cache rpc forever, not sure if it's what we want

Reply:
Good point, thx